lib/model/entity/fiscaat/CiviCategorie.class.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use CsrDelft\Orm\Entity\PersistentEntity;
+use CsrDelft\Orm\Entity\T;
+
+require_once 'model/entity/fiscaat/CiviSaldoCommissieEnum.class.php';
+
+/**
+ * Class CiviCategorie
```

Comment:
Heb je ergens documentatie van de entities? Zo nee, dan mag je dat hier neerzetten. Anders op de GH-wiki ofzo.
De naam van deze entity is me te algemeen.